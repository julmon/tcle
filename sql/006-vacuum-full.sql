SET search_path TO public, s1;
-- Test VACUM FULL
VACUUM FULL t;
SELECT * FROM t WHERE l ILIKE '%updated%';
