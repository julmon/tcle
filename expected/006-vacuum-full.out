SET search_path TO public, s1;
-- Test VACUM FULL
VACUUM FULL t;
SELECT * FROM t WHERE l ILIKE '%updated%';
 p | n |      l       |              d               
---+---+--------------+------------------------------
 3 | 6 | Updated text | Fri Jun 26 22:56:50 2020 PDT
(1 row)

